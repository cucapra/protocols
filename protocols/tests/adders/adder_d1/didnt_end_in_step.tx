// ARGS: --verilog adders/adder_d1/add_d1.v --protocol adders/adder_d1/add_d1.prot
// RETURN: 101
add_doesnt_end_in_step(1, 2, 3);

