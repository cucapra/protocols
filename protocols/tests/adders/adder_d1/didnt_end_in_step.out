error: The last executed statement in Thread 0 (transaction 'add_doesnt_end_in_step') wasn't `step()` (all threads must end their execution with a call to `step()`)
   ┌─ adders/adder_d1/add_d1.prot:28:3
   │
28 │   fork();
   │   ^^^^^^^ last statement wasn't `step()`

